src/types/kubectl.ts

```diff
@@ -3,11 +3,11 @@ import {createInlineArray} from '../utilities/arrayUtils'
 import * as core from '@actions/core'
 import * as toolCache from '@actions/tool-cache'
 import * as io from '@actions/io'
-import {exec} from 'child_process'
 
 export interface Resource {
    name: string
    type: string
+   namespace?: string
```

Comment:
nit: in addition to maybe moving name and namespace into metadata, can we move Resource to k8sobjects.ts?